packages/code/src/Code/Code.spec.tsx

```diff
@@ -237,4 +237,75 @@ describe('packages/Code', () => {
       });
     });
   });
+
+  describe('when expandable', () => {
+    const longCodeSnippet = `
+      const greeting = "Hello, world! 1";
+      const greeting2 = "Hello, world! 2";
+      const greeting3 = "Hello, world! 3";
+      const greeting4 = "Hello, world! 4";
+      const greeting5 = "Hello, world! 5";
+      const greeting6 = "Hello, world! 6";
+    `;
+
+    test('shows no expand button when <= 5 lines of code', () => {
+      render(
+        <Code expandable={true} language="javascript">
+          {codeSnippet}
+        </Code>,
+      );
+
+      expect(screen.queryByRole('button', { name: /expand/i })).toBeNull();
```

Comment:
We should use a testid here. This would also make it easier for consumers to target this button.

Reply:
Good point! I have to get into the design system mindset. I generally try to avoid test ID's to avoid bloating markup for test purposes, but what you're saying makes sense given how consumers might use it. Will update!